session-logout-component.js

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright (c) 2015-2016 Digital Bazaar, Inc. All rights reserved.
+ */
+define(['angular'], function(angular) {
+
+'use strict';
+
+function register(module) {
+  module.component('brSessionLogout', {
+    controller: Ctrl,
+    bindings: {
+      callback: '&?brLogoutCallback'
```

Comment:
Be advised, this is a breaking change which means this would be a semver major release.